pkg/cmd/server/admin/create_keypair.go

```diff
@@ -112,7 +112,7 @@ func (o CreateKeyPairOptions) CreateKeyPair() error {
 		return err
 	}
 
-	fmt.Fprintf(o.Output, "Generated new key pair as %s and %s\n", o.PublicKeyFile, o.PrivateKeyFile)
```

Comment:
Yeah, this is the delegation thing I was talking about.  @sosiouxme requested these as `Printf`s in the case where you're actually trying to run this command.  When you run this command in a delegated case, you should pass an `o.Output` that actual writes to `glog.V(3).Info`.